<launch>

  <env name="LIBGL_ALWAYS_SOFTWARE" value="1"/>

  <!-- read value of arg for configuration file-->
  <arg name="config" default="get_cloud" />
  <param name="param" value="$(arg config)"/> 
  
  <rosparam file="$(find seam_detection)/config/$(arg config).yaml" />
                    
  <node name="rviz" 
    pkg="rviz" 
    type="rviz" 
    args="-d $(find seam_detection)/rviz/save_cloud.rviz">
  </node>

  <node                     
    pkg="seam_detection"          
    name="get_cloud"   
    type="get_cloud"
    output="screen"  
  />
   
</launch>
